WEB/Src/Web/Web.Tests/Web.Tests.csproj

```diff
@@ -33,7 +33,7 @@
 
     <PackageReference Include="Castle.Core" Version="4.4.0" />
     <PackageReference Include="Microsoft.AspNet.Mvc" Version="5.2.4" />
-    <PackageReference Include="Microsoft.AspNet.TelemetryCorrelation" Version="1.0.7" />
+    <PackageReference Include="Microsoft.AspNet.TelemetryCorrelation" Version="1.0.8" />
```

Comment:
will code-less be able to benefit from this by simply updating the version of TelemetryCorrelation package alone?

Reply:
Yes. TelemetryCorrelation module now creates two activity for rewritten requests. Version 1.0.8 generates only one activity for rewritten request.